### ISSUE TITLE
The application screen content is not clearly visible on zooming to 400%.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#115878]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_VSCode;#A11ySev1;#A11yTCS;#A11yWCAG2.1;#DesktopApp;#WCAG1.4.10;#FTP;#Zoom;#Win32;#Visual Studio Code;#CAITestPass_May2020;

### Environment Details:
Application Name: Visual Studio Code
Version: 1.53.0 (user setup)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042

### Pre-requisite: 
Set the display resolution to 1280*1024 and advanced scaling to 400% and sign out and launch the application.

### Repro Steps:
1 . Launch visual studio code application.
2 . For example if the Welcome screen is not shown on the application launch, then navigate to Help --> Welcome.

### Actual:
The application screen content is not clearly visible on zooming the screen content to 400% there is overlapping of contents throughout the application.

### Expected:
All the screen content should be visible clearly when scrolling through the screen or navigating using the tab key. In any given screen the content should not get overlapped, there should be no loss of information.

### User Impact:
Keyboard users will not be able to view all screen content present when the screen is magnified and access the information on activating the controls.

### Attachment for Reference:
![VSCode Reflow](https://user-images.githubusercontent.com/52036772/107033821-d7be3e00-67db-11eb-9779-4cc8146fef1b.png)

**Comment by isidorn** [comment]
I just tried this on my machine and I was able to zoom 400% nicely.
What you might want is for content to get hidden automatically, however that is against vscode UX guidelines - we never hide content automatically and we leave this up to the user.

For example when I hide the panel, the activity bar and the status bar I get a pretty well functioning editor.
Apart from this we never received feedback from the community regarding this and we have quite some users of magnifiers.

Thus we have no plans to change our UI to automatically hide, we leave this up to the user so he or she can choose what is visible.

![Screenshot 2021-02-10 at 18 07 15](https://user-images.githubusercontent.com/1926584/107544741-d1a8d280-6bca-11eb-9d1d-8fdb35b2c90b.png)

**Comment by isidorn** [comment]
For zooming just the editor we suggest to use the following command:
"Editor Font Zoom In" that way the editor content can be zoomed in 400% with none of the other UI being zoomed.
This is actually how Microsoft Word behaves - I just checked.

**Comment by MayuriNaik** [comment]
In order to test reflow in windows, we first need to set the display resolution to 1280*1024 and then change the advanced scaling to 400% and then launch the application. Here the reflow fails for VS Code application and there is an overlapping of contents.

The screenshot that has been provided in the above comment is from a Mac machine where the 'zoom-in' has been performed with the available VS Code keyboard shortcut as 'Ctrl + ='. Here the reflow has not been performed, it's zooming for the application.

The above issue is observed in the windows machine while performing reflow testing, which is still reproducing so this bug should be reopened.

**Comment by isidorn** [comment]
We are in the process of getting an exception for supporting reflow
@sana-ajani

**Comment by sana-ajani** [comment]
Hi, we've been approved for an exception for this.
